source/client-side-operations-timeout/tests/runCursorCommand.yml

```diff
@@ -0,0 +1,316 @@
+description: runCursorCommand
+
+schemaVersion: '1.9'
+
+runOnRequirements:
+  - minServerVersion: "4.4"
+
+createEntities:
+  - client:
+      id: &failPointClient failPointClient
+      useMultipleMongoses: false
+  - client:
+      id: &commandClient commandClient
+      useMultipleMongoses: false
+      observeEvents: [commandStartedEvent, commandSucceededEvent]
+  - client:
+      id: &client client
+      useMultipleMongoses: false
+      observeEvents: [commandStartedEvent]
```

Comment:
We should ignore killCursor events for the client. This conflicts with "Non-tailable cursor lifetime remaining timeoutMS applied to getMore if timeoutMode is unset".

```
    unified_spec_runner.go:134: events verification failed at index 0: extra events published; all events for client:

        Started Events

        [localhost:27017[-17]] {"find": "collection","batchSize": {"$numberInt":"1"},"lsid": {"id": {"$binary":{"base64":"9ydWCZL/Q+i6PhkMWPGjDg==","subType":"04"}}},"$clusterTime": {"clusterTime": {"$timestamp":{"t":1684442140,"i":28}},"signature": {"hash": {"$binary":{"base64":"AAAAAAAAAAAAAAAAAAAAAAAAAAA=","subType":"00"}},"keyId": {"$numberLong":"0"}}},"maxTimeMS": {"$numberLong":"56"},"$db": "db"}
        [localhost:27017[-17]] {"getMore": {"$numberLong":"6334217656449035453"},"collection": "collection","lsid": {"id": {"$binary":{"base64":"9ydWCZL/Q+i6PhkMWPGjDg==","subType":"04"}}},"$clusterTime": {"clusterTime": {"$timestamp":{"t":1684442140,"i":28}},"signature": {"hash": {"$binary":{"base64":"AAAAAAAAAAAAAAAAAAAAAAAAAAA=","subType":"00"}},"keyId": {"$numberLong":"0"}}},"maxTimeMS": {"$numberLong":"9"},"$db": "db"}
        [localhost:27017[-18]] {"killCursors": "collection","cursors": [{"$numberLong":"6334217656449035453"}],"lsid": {"id": {"$binary":{"base64":"9ydWCZL/Q+i6PhkMWPGjDg==","subType":"04"}}},"$clusterTime": {"clusterTime": {"$timestamp":{"t":1684442140,"i":28}},"signature": {"hash": {"$binary":{"base64":"AAAAAAAAAAAAAAAAAAAAAAAAAAA=","subType":"00"}},"keyId": {"$numberLong":"0"}}},"$db": "db"}
```

Reply:
Agreed, and this aligns with other CSOT tests, updated